src/components/atomic/Background/Neons/Neons.styled.tsx

```diff
@@ -0,0 +1,15 @@
+import { styled } from '@mui/material/styles';
+import { ReactComponent as NeonBox1Svg } from './neon1.svg';
+import { ReactComponent as NeonBox2Svg } from './neon2.svg';
+
+export const NeonBox1 = styled(NeonBox1Svg)`
```

Comment:
what if we name them "NeonBoxInner" and "NeonBoxOuter" for readability?

Reply:
sure can work better!

Reply:
fixed in: 45b92be513a92659393013afe709b0706e64af8e